src/Tester.java

```diff
@@ -59,15 +59,18 @@ private static void run_test_case(Class test_case_class) {
             throw new RuntimeException("Check BeforeSuite or AfterSuite annotated methods. They must be in a single implementation.");
         }
 
+        System.out.printf("Running %s\n", test_case_class);
```

Comment:
Our static analysis engine detects that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.